elasticsearch/elasticsearch.go

```diff
@@ -63,7 +66,7 @@ func (b *BulkIndexer) Add(r elastic.BulkableRequest) error {
 }
 
 func (b *BulkIndexer) Flush() error {
-	_, err := b.bulkService.Do()
+	_, err := b.bulkService.Do(context.TODO())
```

Comment:
These contexts _may_ become useful at some point in the future. 
Good to know that is available.